src/styles/app.scss

```diff
@@ -96,3 +96,7 @@
 #release-head {
   font-family: 'Roboto', sans-serif;
 }
+
+.dropdown-menu {
+  min-width: 100px !important;
```

Comment:
@Rhotimee Consider using a new class to style the button since `.dropdown-menu` might be reusable which might affect a different dropdown too.

Reply:
Done